### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by mrbhi**
Added the desktop version of my portfolio website.
Updated my README file.

Thanks for your PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,20 +209,28 @@ <h2>Skills</h2>
...
+          <form action="https://formspree.io/f/meqdgvvo" method="post" id="contact-form">
+            <input type="text" placeholder="Full name" name="name" maxlength="30" required>
+            <input type="email" placeholder="Email address" name="email" required>
+            <textarea name="message" id="msg" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button type="submit" class="submit">Get in touch</button>
+          </form> 
+        </div>
       </div>
       <footer class="social-link">
         <hr class="line-2">
```
Comment (Gopxfs) [inline index.html:237]: Please adjust your footer line to correspond to the Figma design. It should be slightly thinner and with a 100% width.

Figma design:
image

Your design:
image

File: css/style.css [hunk 2]
```diff
@@ -532,9 +547,712 @@
...
+  .submit:active {
+    background-color: #f55800;
+  }
+
+  .submit:disabled {
+    background-color: #f4f1f0;
+  }
+}
+
+@media screen and (min-width: 768px) {
```
Comment (Gopxfs) [inline css/style.css:562]: Currently, you have one media query for max-width: 500px and one for min-width: 768px. This means that your webpage _doesn't_ contain any styling between 500px and 768px. Please adjust the first query to max-width: 768px to fix this problem and display your beautiful work on all devices. 😄

Between 500~768px:
size-500-768

File: index.html [hunk 3]
```diff
@@ -86,18 +95,18 @@ <h1>Hey There. <br />I’m Alex</h1>
...
       </div>
       <div class="frame-container">
-        <div class="card-works">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-          <ul>
+        <div id="work1" class="card-works">
+          <h2 class="project">Profesional Art Printing Data</h2>
+          <p class="project">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="project">
```
Comment (Gopxfs) [inline index.html:101]: [OPTIONAL] A tip for you: I've noticed that you made the first card different from the others, just like in the design:
image

If you pay closer attention, you will see that in the design there is a cursor hovering over this first card:
image
This means that this card isn't different from the other ones, the design is showing to you the _hover_ effect of the cards. 😉
In short, all your cards should have the title/description but, when hovered, the project picture appears clearly and the _See Project_ button is displayed.
Keep up the good work! Your project look amazing.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @mrbhi 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ Nice usage of grid & flexbox for the page structure!
✔️ Beautiful CSS implementation.
✔️ Your project is very well done, keep up the good work!
> TIP: Although your readme looks really good, it would be nice to put your project title instead of 'Project Name' followed by a brief explanation of what you made. 😉 Good luck!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @mrbhi, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats


## Good Highlights✔

- You have implemented suggested changes
- Linter checks are passing
- Descriptive PR title


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**